packages/core/content-manager/shared/contracts/history-versions.ts

```diff
@@ -43,7 +44,7 @@ export declare namespace GetHistoryVersions {
     };
     query: {
       contentType: UID.ContentType;
-      documentId: Entity.ID;
+      documentId?: Entity.ID;
```

Comment:
For single types the frontend doesn't have the document ID, the backend has to find it, which is the fix I was trying to implement but pushed off because of several bugs with single types